shared/js/background/chrome-events.es6.js

```diff
@@ -216,6 +216,29 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
     }
 })
 
+/**
+ * Chrome "Change back to Google Search" dialogue detection
```

Comment:
Maybe scope this to chome(-ium?) browsers only?

Reply:
Thanks @jonathanKingston I first naively thought that `chrome-events.es6.js` was chrome specific but realized that it is not the case!